.drone.yml

```diff
@@ -44,6 +44,11 @@ matrix:
     - { TARGET: test, CONSTEL: ui.grid }
     - { TARGET: test, CONSTEL: ui.scheduler }
     - { TARGET: test, CONSTEL: viz }
+    - { TARGET: test, IOS: true, JQUERY: true, CONSTEL: ui,  }
```

Comment:
What about "export" constellation?

Reply:
These tests are stable on the farm. So, if the problems occurs - we will add them too.